tsnet/depaware.txt

```diff
@@ -123,6 +123,7 @@ tailscale.com/tsnet dependencies: (generated by github.com/tailscale/depaware)
         github.com/klauspost/compress/fse                            from github.com/klauspost/compress/huff0
         github.com/klauspost/compress/huff0                          from github.com/klauspost/compress/zstd
         github.com/klauspost/compress/internal/cpuinfo               from github.com/klauspost/compress/huff0+
+     💣 github.com/klauspost/compress/internal/le                    from github.com/klauspost/compress/huff0+
```

Comment:
Not sure what the 💣 means here. @tomhjp @irbekrm ?

Reply:
(We do want to add all these depaware changes to the PR/commit header though)